src/index.js

```diff
@@ -65,7 +66,8 @@ Cache.prototype.retrieve = function (key, options, callback = noop) {
 }
 
 Cache.prototype.createStream = function (key, options = {}) {
-  return h(this.store.get(key))
+  const features = this.store.get(key)
+  return Readable.from(features)
```

Comment:
I take it you don't prefer the one-liner here?

Reply:
Right.  One liners are less characters to type, but often take longer to read, and I spend more time reading code than writing it.  So over the lifetime of this code, I think two lines here will speed comprehension and save developers time.  Also, it make it self-documenting, i.e., `features` array is what is coming back from the store and being streamed.

Reply:
Ahh, yea the self-documenting definitely key. Didn't think of that.